styles/globals.css

```diff
@@ -63,6 +63,7 @@ footer a {
 
 .main-container {
   height: 100%;
+  
```

Comment:
I am not sure why there is an extra space here

Reply:
which space are you talking about? @Harshi7016

Reply:
Extra line in globals.css , this can be  ignored

Reply:
O will be removed 👍